webassembly/testing/CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+set(f3djsTests_list
+  "test_options"
+)
+
+# Add all the ADD_TEST for each test
+foreach(test_name ${f3djsTests_list})
+
+  set(TEST_JS_MODULE "${CMAKE_CURRENT_SOURCE_DIR}/${test_name}.js")
+  configure_file(
+    "${CMAKE_CURRENT_SOURCE_DIR}/test.html.in"
+    "${CMAKE_CURRENT_BINARY_DIR}/${test_name}.html"
+    @ONLY)
+
+  add_test(NAME f3djs::${test_name}
+    COMMAND chromium
```

Comment:
`find_executable(chromium)` somewhere seems needed.

Reply:
not strictly needed because it's in the Docker image for sure, but I've added this

Reply:
ok i got it now.